task5/servlets/Check.java

```diff
@@ -0,0 +1,16 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class Check extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
This method overrides the `doPost` method from the superclass, therefore, must be annotated with `@Overrides`